app/src/main/java/a/a/a/BuiltInLibraryUtils.java

```diff
@@ -0,0 +1,59 @@
+package a.a.a;
+
+import mod.jbk.build.BuiltInLibraries;
+import mod.jbk.build.BuiltInLibraries.BuiltInLibrary;
+
+public class BuiltInLibraryUtils {
+
+    /**
+     * Returns the known dependencies for a given built-in library.
+     *
+     * @param libraryName The name of the built-in library.
+     * @return An array of known dependencies for the library.
+     * @apiNote This method won't return the dependencies' sub-dependencies.
+     * @throws IllegalArgumentException If the input name doesn't match any known built-in library.
+     */
+    public static String[] getKnownDependencies(String libraryName) {
+        for (BuiltInLibrary library : BuiltInLibraries.KNOWN_BUILT_IN_LIBRARIES) {
+            if (library.getName().equals(libraryName)) {
+                return library.getDependencyNames().toArray(new String[0]);
+            }
+        }
+
+        throw new IllegalArgumentException("Unknown built-in library '" + libraryName + "'!");
+    }
+
+    /**
+     * Returns the package name of a given built-in library.
```

Comment:
over-documenting simple methods like these seems mostly unnecessary to me; either `@return` or the javadoc body is redundant since they have the same meaning